src/pages/IndexPage.vue

```diff
@@ -216,6 +216,7 @@ generic-panel(
         target="_blank"
         no-caps
         unelevated
+        class="btn-contribute"
```

Comment:
Please use the pug-style class usage for static css classes. Like so:

Use `component(....).btn-contribute` over `component(... class="btn-contribute")` 

Just a small fix then we are good to go! 💪

Reply:
> Please use the pug-style class usage for static css classes. Like so:
> 
> Use `component(....).btn-contribute` over `component(... class="btn-contribute")`
> 
> Just a small fix then we are good to go! 💪

Sure! Will do it asap!